builds/run_sbt_tests.sh

```diff
@@ -34,7 +34,7 @@ echo "*** Running sbt tests"
 
 if [[ -f "$PROJECT_DIRECTORY/docker-compose.yml" ]]
 then
-  ./builds/run_sbt_task_in_docker.sh "project $PROJECT_NAME" "dockerComposeTest"
```

Comment:
I think we should.  I don't know what it is about this command that breaks discovery, because it does find some tests, just not all of them.   If these other projects have tests with whatever characteristic means they can't be found by dockerComposeTest, then we could have quite a widespread problem.

Reply:
🤞 

https://github.com/wellcomecollection/scala-libs/pull/197
https://github.com/wellcomecollection/storage-service/pull/1038
https://github.com/wellcomecollection/catalogue-api/pull/593